src/components/cardbuilder/card.css

```diff
@@ -174,6 +174,7 @@ button::-moz-focus-inner {
 
 .cardScalable .cardImageContainer {
     height: 100%;
+    width: 100%;
```

Comment:
This doesn't seem to affect anything is it needed?

Reply:
It definitely fixed some issues with the layout when I was testing, are you on Firefox or another browser?

Reply:
Firefox

Reply:
I've confirmed this fixes an issue on my browser.